napari/layers/points/_tests/test_points_utils.py

```diff
@@ -1,12 +1,29 @@
 import numpy as np
 import pandas as pd
 
-from napari.layers.points.points_utils import (
+from napari.layers import Points
+from napari.layers.points._points_utils import (
     dataframe_to_properties,
     guess_continuous,
 )
 
 
+def create_known_points_layer():
+    """Create points layer with known coordinates."""
+    data = [[1, 3], [8, 4], [10, 10], [15, 4]]
+    first_point = [1, 3]
+    known_non_point = [20, 30]
```

Comment:
I think it would be nice to have a comment explaining what `known_non_point` is. There could be enough in this function to warrant a full docstring explaining the returned variables.

Reply:
good point, doc string added